Docs fixes arising from testing the local deployment workflow for mobile. (#723)
spatialos/gdk-for-unity: docs/content/get-started/set-up.md

 **Using Visual Studio?**

> As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find this by clicking the **Modify** option for Visual Studio Community 2017. If the **Modify** option is not visible, this option may also be found in the drop-down menu that appears when you select the **More** option.

Reviewer: doesn't have to be the community version :)

also: I have never seen that modify option, I wonder which option we should show first as I suspect that most people will not read the last sentence ;)

Author: ![capture](https://user-images.githubusercontent.com/19690292/52212823-539bed80-2885-11e9-8cda-af15cbed6d16.PNG)

Good shout. I've removed "Community", tweaked some language and added bullet-points in attempt to give this section more of an obvious though line / call to action.

 **Using Visual Studio?**

You need to install the **.NET Core cross-platform development** and **Game development with Unity** workloads. To to this:

> - As you install [Visual Studio](https://www.visualstudio.com/downloads/), select the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find the **Workloads** tab by opening Visual Studio Installer and, in the **Products** section, selecting **Modify** for Visual Studio 2017. If you can't see the **Modify** option, select **More**.

Reviewer 2: I did an edit on this in Jamie's branch too!

> - As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find the **Workloads** tab by opening Visual Studio Installer and, in the Products section, clicking the **Modify** option for Visual Studio 2017. If the **Modify** option is not visible, this option may also be found in the drop-down menu that appears when you select the **More** option.

Reviewer 2: If "Products" is part of the UI, it's bolded.

```suggestion
- As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find the **Workloads** tab by opening Visual Studio Installer and, in the **Products** section, selecting **Modify** for Visual Studio 2017. If you can't see the **Modify** option is not visible, select the **More**.
```
"Select" not "click on" or "click".
Avoid gerunds where possible and use imperatives (instructions).
